src/main/java/com/kodilla/ecommercee/dao/ProductDao.java

```diff
@@ -7,20 +7,13 @@
 import javax.transaction.Transactional;
 import java.math.BigDecimal;
 import java.util.List;
-import java.util.Optional;
 
 @Transactional
 @Repository
 public interface ProductDao extends CrudRepository<Product, Long> {
-    @Override
-    List<Product> findAll();
 
     @Override
-    Optional<Product> findById(Long productId);
```

Comment:
Nie powinno być szukania i usuwanie produktu po numerze id?

Reply:
czemu, usuwanie i szukanie to metody domyślne interfejsu CrudRepository, nie trzeba ich nadpisywać jeżeli mają zdefiniowane obiekty na których operują (linia 13), po prostu się ją stosuje. W przypadku findAll() interfejs zwraca Iterable<Product> więc tu zmieniliśmy to na List które dziedziczy po Iterable więc jest @Override